resources/imdb_data.csv

```diff
@@ -0,0 +1,7 @@
+﻿item,rating,type
+space jam,9,movie
```

Comment:
Is this a realistic sample of the imdb data? Any idea on if we will have more data? If so might cause future problems if we're just accessing an index in a row if anything changes. Maybe add an issue so we're aware?